.gitignore

```diff
@@ -54,8 +54,8 @@ typings/
 # Yarn Integrity file
 .yarn-integrity
 
-# dotenv environment variables file
-.env
```

Comment:
@supachris28, why has this been removed?

Reply:
There is no runtime for this, it is a module, so no real env variables to be used. I've added a `.env` to be used for testing to ensure it works as expected.